tom_demoapp/apps.py

```diff
@@ -16,8 +17,8 @@ def target_detail_buttons(self):
         - 'class': The CSS class of the button
         - 'text': The text of the button
         """
-        return {'namespace': 'demoapp:demo-page',
-                'title': 'Demo Target Button',
+        return {'namespace': f'{self.label}:demo-page',
+                'title': f'{self.label} Target Button',
```

Comment:
This will likely NOT be what you want in most Apps. I think it would be better to leave the title without a reference to `self.label`

Reply:
I think this is better fixed by changing the structure of this integration point to match the partial+context strategy used below.

Reply:
I agree that the button title will have to be changed, but a button initially titled "my_app Target Button is better than "Demo Target Button" because it (again, initially, until the TOM dev changes it) reflects the app the button originates from.

Reply:
the problem with doing that now is that it would be a breaking change now :(